ui/app/AppLayouts/Chat/panels/communities/CommunityWelcomeBannerPanel.qml

```diff
@@ -105,4 +105,8 @@ Rectangle {
             community: root.activeCommunity
         })
     }
+
+    Component.onCompleted: {
+        localAccountSensitiveSettings.hiddenCommunityWelcomeBanners = [];
```

Comment:
@alexandraB99 cases like this and others will overwrite a value read from the local file.

Reply:
removed